api-doc/openapi.yaml

```diff
@@ -1014,6 +1110,70 @@ components:
         example:
           id: 3a896646-70e0-4943-a3e3-01f29d0541f1
           parameter: kalle tetr
+    Timer:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        start_time:
+          type: string
+          format: date-time
+        end_time:
+          type: string
+          format: date-time
+      required:
+        - id
+        - timestamps
+      example:
+        id: 91aee618-2c62-42a1-b025-106baf6b71e7
+        name: ykköstimeri
+        start_time: '2023-09-21T15:05:47.000+00:00'
+        end_time: '2023-09-21T15:05:48.000+00:00'
+    TimerInput:
+      type: object
+      properties:
+        name:
+          type: string
+        start_time:
+          type: string
+          format: date-time
+          nullable: true
+        end_time:
+          type: string
+          format: date-time
+          nullable: true
+      required:
+        - start_time
+        - end_time
```

Comment:
Not required I think? Or did you have something specific in mind with nullable required field?

```suggestion
```

Reply:
Not required